redis/src/streams.rs

```diff
@@ -34,6 +34,59 @@ impl ToRedisArgs for StreamMaxlen {
     }
 }
 
+/// Builder options for [`xadd_options`] command.
+///
+/// [`xadd_options`]: ../trait.Commands.html#method.xadd_options
+///
+#[derive(Default, Debug)]
+pub struct StreamAddOptions {
+    /// Set MAXLEN <threshold> cmd arg.
+    maxlen: Option<StreamMaxlen>,
```

Comment:
I think we should use snake_case names (`max_len`, `min_id`, `no_mk_stream`) for both the fields and the methods here.